examples/airsim/worldGen.scenic

```diff
@@ -1,39 +1,38 @@
-# NOTE: add your world info path here
-# param worldInfoPath = "[YOUR PATH HERE]"
-
-model scenic.simulators.airsim.model
-from scenic.simulators.airsim.utils import getPrexistingObj
-import random
-
-ground = getPrexistingObj("ground")
-
-ground.highlight()
-
-
-ego = new StaticObj on ground,
-    with assetName "Cone",
-    with width 10,
-    with length 10,
-    with height 10
-
-
-centerArea = RectangularRegion(Vector(0,200,30), 0, 100,100)
-
-blocks = []
-blockCount = 10
-
-
-
-for i in range(blockCount):
-    blocks.append(new StaticObj on ground, 
-        contained in centerArea,
-        with assetName "Cube",
-        with width Range(3,10),
-        with length Range(3,10),
-        with height 10)
-    
-
-ranBlock = blocks[random.randint(0,blockCount-1)]
-
-drone = new Drone on ranBlock, with behavior Patrol([(10,10,10),(2,2,2)])
-
+# NOTE: add your world info path here
+# param worldInfoPath = "[YOUR PATH HERE]"
+
+model scenic.simulators.airsim.model
+from scenic.simulators.airsim.utils import getPrexistingObj
+import random
+
+ground = getPrexistingObj("ground")
+
+ground.highlight()
+
+
+ego = new StaticObj on ground,
+    with assetName "Cone",
+    with width 10,
+    with length 10,
+    with height 10
+
+
+centerArea = RectangularRegion(Vector(0,200,30), 0, 100,100)
+
+blocks = []
+blockCount = 10
+
+
+
+for i in range(blockCount):
+    blocks.append(new StaticObj on ground, 
+        contained in centerArea,
+        with assetName "Cube",
+        with width Range(3,10),
+        with length Range(3,10),
+        with height 10)
+    
+
+ranBlock = blocks[random.randint(0,blockCount-1)]

```

Comment:
This won't work (you can't use `random` in the normal way in a Scenic scenario). I'd try `ranBlock = Uniform(*blocks)`.